cocos/physics-2d/box2d-wasm/joints/spring-joint.ts

```diff
@@ -56,8 +50,8 @@ export class B2SpringJoint extends B2Joint implements ISpringJoint {
         def.localAnchorA = { x: comp.anchor.x / PHYSICS_2D_PTM_RATIO, y: comp.anchor.y / PHYSICS_2D_PTM_RATIO };
         def.localAnchorB = { x: comp.connectedAnchor.x / PHYSICS_2D_PTM_RATIO, y: comp.connectedAnchor.y / PHYSICS_2D_PTM_RATIO };
         def.length = comp.distance / PHYSICS_2D_PTM_RATIO;
-        def.damping = 0;//comp.dampingRatio;
-        def.stiffness = 1;//comp.frequency;
```

Comment:
Why was it hard-coded before?

Reply:
it will then be set here
![image](https://github.com/cocos/cocos-engine/assets/9999378/44ac18ea-8109-4b48-b869-81b86b166f28)